src/app.js

```diff
@@ -90,17 +91,19 @@ async function run() {
 
 	// Check Mongo for jobs
 	async function checkMongo() {
+		let correlationId;
```

Comment:
Don't use `let`. Catch the error in inner block after `correlationId` has already been declared (as const). The outer block can still have the try-catch (If needed) but it will catch more general errors (Which won't need `correlationId` since it's already handled by inner try-catch)